cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=True)
+    duration = serializers.IntegerField()
```

Comment:
what is different between 
`serializers.CharField(required=True)` and `serializers.CharField()`

Reply:
required=True it's default value, I will remember it